src/components/Skeleton/Skeleton.tsx

```diff
@@ -41,8 +41,7 @@ export const useStyles = makeStyles(
 
       '&::after': {
         animation: '$shine 1.6s linear 0.5s infinite',
-        background:
-          'linear-gradient(90deg, transparent, rgba(230, 231, 237, 1), transparent)', // graphite[200] with opacity
+        background: `linear-gradient(90deg, transparent, ${theme.palette.graphite[100]}, transparent)`,
```

Comment:
From the comment, it looks like it should be `theme.hexToRgba(theme.palette.graphite[200], 1)`, is that no longer the case?

Reply:
I can change it to that if you prefer! Here's why it is the way it is: 

- (I just doublechecked) Those values are #e6e7ed, or graphite[100], so I'm guessing it was changed or mislabelled before
- With an alpha of 1, I decided to skip the theme.hexToRgba() part. But if we think this may need alpha in the future and want to decrease cognitive load on that future developer, I'd be happy to change it back.